### DISCUSSION TITLE
a working example for whisper.py?

### DISCUSSION THREAD

**Discussion by gonenraveh** [tinygrad/tinygrad#1860] [Q&A]
I have tested this:
python whisper.py tests_jfk.flac

And I get an error:
AssertionError: Input Tensor shape (80, 1515) does not match the shape of the weights (768, 80, 3). (80 vs. 1515)

Please advise,
Gonen

**Comment by wozeparrot** [comment]
There is a WIP pr here to clean up whisper: #1410, it has somewhat stalled, but it should still function.
